web_src/css/base.css

```diff
@@ -1285,7 +1285,14 @@ a.ui.card:hover,
 .following.bar #navbar {
   width: 100vw;
   min-height: 52px;
-  padding: 0 0.5rem;
+  padding: 0 16px;
+}
+
+@media (max-width: 767px) {
+  .following.bar #navbar {
+    padding-left: 4px;
```

Comment:
just curious why 4px is added to the padding-left, the visual effect looks good though.

Reply:
It's just a pixel hack on top of fomantic's silly ` .92857143em` padding inside the box 😉.

Reply:
lol I see.